package.json

```diff
@@ -14,6 +14,9 @@
     "dev:frontend": "nuxt",
     "build": "nuxt build --modern=client",
     "build:analyze": "nuxt build -a --modern=client",
+    "lhci:mobile": "lhci autorun",
```

Comment:
What is it?

Reply:
This is the script for running Lighthouse CI with configuration from `lighthouserc.json`. We're starting use this tool as a part of our ci process.